### ISSUE TITLE
Docs for "Configuring cross-origin resource sharing" contains incorrect commands

### ISSUE THREAD

**Issue by agajdosi** [minishift/minishift#881]
**Description**
Command variants of `minishift openshift config set --patch '{"corsAllowedOrigins": [".*"]}'` for CMD and PowerShell in [Example: Configuring cross-origin resource sharing](https://docs.openshift.org/latest/minishift/using/interacting-with-openshift.html#example-config-cors) are unrelated to the topic of this chapter. This is probably caused because when we were finding solution for both PowerShell and CMD we switched for more complicated command which then got inserted into documentation.

**Actual commands:** 
```
PowerShell:
.\minishift.exe --% openshift config set --patch "{\"routingConfig\": {\"subdomain\": "192.168.99.101.nip.io\"}}"

CMD:
minishift.exe openshift config set --patch "{\"routingConfig\": {\"subdomain\": \"192.168.99.101.nip.io\"}}"
```
**Expected:**
Commands listed as fix for Windows CMD and PowerShell should set corsAllowedOrigins.

**Tested solution:**
```
CMD - Windows 7, Windows 10:
minishift openshift config set --patch "{\"corsAllowedOrigins\": [\".*\"]}"

PowerShell - Windows 7, Windows 10:
minishift openshift config set --patch '{\"corsAllowedOrigins\": [\".*\"]}'
```

**Comment by agajdosi** [comment]
Tested this on Windows 10, variants for CDM and PowerShell are here the same as in Windows 7.
```
for CMD:
minishift openshift config set --patch "{\"corsAllowedOrigins\": [\".*\"]}"

for PowerShell:
minishift openshift config set --patch '{\"corsAllowedOrigins\": [\".*\"]}'
```

**Comment by Preeticp** [comment]
Closing this issue as it has been resolved by PR #918

**Comment by jorgemoralespou** [comment]
@hferentschik this should be provided by an addon given the incidence that this can have (frequent).
